cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -126,8 +149,18 @@ func (t *templateBuilder) buildNodeFromTemplate(mig *Mig, template *gce.Instance
 				return nil, err
 			}
 			node.Spec.Taints = append(node.Spec.Taints, kubeEnvTaints...)
+
+			if allocatable, err := t.buildAllocatable(node.Status.Capacity, *item.Value); err == nil {
```

Comment:
what if error is not null? should we print something?

Reply:
This code assumes there can be multiple kube-env entries and accumulates it. If there is one error here it doesn't mean there is no allocatable at all. If this is the case, we log the Warning below. Downside is it doesn't log the error.